cmd/juju/commands/switch.go

```diff
@@ -47,7 +50,7 @@ default model in mycontroller, :mymodel switches to mymodel in current
 controller and mycontroller:mymodel switches to mymodel on mycontroller.
 The `[1:] + "`juju models`" + ` command can be used to determine the active model
 (of any controller). An asterisk denotes it.
-
+The special arguments - (hyphen) allows to return to previous model or controller.
```

Comment:
```
Details:
When used without an argument, the command shows the current controller
and its active model.
When a single argument without a colon is provided juju first looks for a
controller by that name and switches to it, and if it's not found it tries
to switch to a model within current controller. mycontroller: switches to
default model in mycontroller, :mymodel switches to mymodel in current
controller and mycontroller:mymodel switches to mymodel on mycontroller.
The `juju models` command can be used to determine the active model
(of any controller). An asterisk denotes it.
The special arguments - (hyphen) allows to return to previous model or controller.
```

This doesn't look very nice. While you're here, could you add some white space?

Reply:
Yes I can. Maybe i can refresh the whole text likewise:
@jack-w-shaw 

```sh
Details:
When used without an argument, the command shows the current controller
and its active model.

When a single argument without a colon is provided juju first looks for a
controller by that name and switches to it, and if it's not found it tries
to switch to a model within current controller. 

Colon allows to disambiguate model over controller:
- mycontroller: switches to default model in mycontroller, 
- :mymodel switches to mymodel in current controller 
- mycontroller:mymodel switches to mymodel on mycontroller.

The special arguments - (hyphen) instead of a model or a controller allows to return 
to previous model or controller.

The `juju models` command can be used to determine the active model
(of any controller). An asterisk denotes it.

```